libs/core/facets/facet/facet.component.html

```diff
@@ -2,7 +2,16 @@
     <ng-template [ngTemplateOutlet]="customContentTemplate"></ng-template>
 } @else {
     @if (facetTitle) {
-        <span fd-title [headerSize]="5" class="fd-margin-bottom--sm" [id]="titleId">{{ facetTitle }}</span>
+        <span
+            [attr.role]="!!headingLevel ? 'heading' : null"
+            [attr.aria-level]="headingLevel"
+            style="display: block"
```

Comment:
Can't we just use `H2` element? it will reduce the coding, no?

Reply:
No, the heading level needs to be configurable by the developer

Reply:
Then can we fix this issue first in styles and then bring it here, instead of having inline styling?